### DISCUSSION TITLE
Any way to optimize the 2008-2011 layouts for dark mode?

### DISCUSSION THREAD

**Discussion by MesaBlack** [lightbeam24/CustomTube#169] [Q&A] [answered]
![image](https://github.com/lightbeam24/CustomTube/assets/132141735/c86e5e03-67e2-47a8-8cb5-789cf066fedd)
(taken on 2008 layout, w/ appearance: dark mode)

![Screenshot 2024-02-01 6 35 49 PM](https://github.com/lightbeam24/CustomTube/assets/132141735/70e225d9-595e-4776-bb19-ccb596944d97)
2009 layout

![Screenshot 2024-02-01 6 36 09 PM](https://github.com/lightbeam24/CustomTube/assets/132141735/b716cdec-22c1-4421-99f1-d0f9f1e01cbc)
2011 layout

**Comment by lightbeam24** [comment, marked as answer]
Yes however it is very low priority at the moment.
